src/model/Text.cpp

```diff
@@ -65,9 +75,10 @@ void Text::scale(double x0, double y0, double fx, double fy) {
     this->font.setSize(size);
 
     this->sizeCalculated = false;
+    this->updateSnapping();
```

Comment:
call calculateSize before this line or remove it, since the width and height is not calculated yet.